### ISSUE TITLE
Deprecate (bool * string) seq overload of classes?

### ISSUE THREAD

**Issue by cmeeren** [Zaid-Ajaj/Feliz#158]
Since we have implicit yields, isn't it just as well to rely on that built-in method and do

```f#
style.classes [
  if x then "foo"
  "bar"
  "baz"
]
```

instead of 

```f#
style.classes [
  x, "foo"
  true, "bar"
  true, "baz"
]
```

If this is changed, remember to update [this page](https://zaid-ajaj.github.io/the-elmish-book/#/chapters/elm/using-css).

**Comment by Zaid-Ajaj** [comment]
That sounds like a good idea :100: one more thing to cleanup before the stable release

**Comment by cmeeren** [comment]
👍 Perhaps it could be good to take a look at all the props and components "one last time" to make sure there will be no superfluous overloads like this left in 1.0.

**Comment by Zaid-Ajaj** [comment]
Sure thing! I will get back to the stable release issue soon and mop up these small API changes

**Comment by Zaid-Ajaj** [comment]
Fixed in v0.76.0 within [this commit](https://github.com/Zaid-Ajaj/Feliz/commit/7da52abd9b59ae4ada3bd100b43647a825b482e3)
